### DISCUSSION TITLE
Is it possible to use for particular project in org.

### DISCUSSION THREAD

**Discussion by Priyanka7850** [github-community-projects/issue-metrics#333] [General]
I'm currently working on analyzing issue metrics for sample organization projects. I've noticed that while each project is spread across multiple repositories, I can only access comprehensive issue metrics for one particular repository. Unfortunately, I don't get issues in other repositories which are also part of the same projects, and also get all the issues it won't get as per the project.

 If you have suggestions on how to move forward, I'd appreciate your guidance on this.

**Comment by zkoppert** [comment]
Yes, this is actually possible!! Check out the [documentation on search filtering for issues related to a project](https://docs.github.com/en/search-github/searching-on-github/searching-issues-and-pull-requests#search-by-project). That will inform your crafted `SEARCH_QUERY` variable when running the issue-metrics action. for example:
`SEARCH_QUERY: 'repo:owner/repo is:issue project:github/57'`
